src/gnutls-2-fixes2.patch

```diff
@@ -0,0 +1,20 @@
+--- a/libdane/dane-params.c	2016-07-12 18:03:22.691584000 +0200
```

Comment:
Please format patches using patch-tool-mxe. See https://github.com/mxe/mxe/pull/1277#issuecomment-257083463 for explanation. Provide commit messages with reasoning of patches.